app/browser/api/ledger.js

```diff
@@ -2362,8 +2362,8 @@ const yoDawg = (stateState) => {
   return stateState
 }
 
-const checkBtcBatMigrated = (state, status) => {
-  if (!status) {
+const checkBtcBatMigrated = (state, paymentsEnabled) => {
+  if (!paymentsEnabled) {
```

Comment:
renamed the variable to be more meaningful (and also match other places in the file)